src/all/koharu/src/eu/kanade/tachiyomi/extension/all/koharu/Koharu.kt

```diff
@@ -28,19 +28,20 @@ import uy.kohesive.injekt.injectLazy
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class Koharu : HttpSource(), ConfigurableSource {
-    override val name = "SchaleNetwork"
+class Koharu(
+    override val lang: String = "all",
+    private val searchLang: String = "",
+    override val id: Long = 2385208600973708857,
+) : HttpSource(), ConfigurableSource {

```

Comment:
Hello, can I just use `override val id = if (lang == "en") 1484902275639232927 else super.id` instead of converting the Koharu class to an abstract class? Thank you.